### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by pachonk**

### CODE REVIEW

**Comment by tmornini** [comment]
@snuggs ???

**Comment by snuggs** [comment]
Fixes #98

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
```
Comment (brandondees) [inline CODE_OF_CONDUCT.md]: I'm partial to keeping it as simple as possible ('BE KIND') and minimizing the emphasis on negatives as well. This boilerplate CoC is really built for projects and organizations of a different scale, which should be considered at the appropriate time in the life of the project, but thoughtfully and with agreement from the existing contributing team.

Comment (snuggs) [inline CODE_OF_CONDUCT.md]: Agreed. Just need @tmornini's blessing and can do the PR with `CODE_OF_CONDUCT.md` at root i'd assume.

Fan of "Be Kind" myself. If we can't do that at this level might as well wrap it up. lol

**Comment by mrbernnz** [comment]
@pachonk @snuggs @janz93 could this be merged in with #147?

**Comment by snuggs** [comment]
@mrbernnz it should have perhaps been merged in with #147. I'm indifferent to them but this is a community project. @mrbernnz @pachonk i feel we have enough input from @janz93 @brandondees and @tmornini. Can we get this one either merged in or closed? starting to gt a little long in the tooth.

**Comment by pachonk** [comment]
The tone of this PR has changed dramatically and I'm not sure it's the same as the original thought into the PR.
I snagged the GitHub recommend Code of Conduct because I wanted to ensure this project had a code of conduct. It is based on the http://contributor-covenant.org code of conduct which is fairly widely used as a sane default code of conduct.
Being that it sounds like we don't want to utilize this http://contributor-covenant.org code of conduct, I'm closing this PR.

**Pull request closed without merging**